packages/blockchain-wallet-v4-frontend/src/modals/Onboarding/KycVerification/UserInfoDetails/UserInfoDetails.template.tsx

```diff
@@ -0,0 +1,210 @@
+import React from 'react'
+import { FormattedMessage } from 'react-intl'
+// @ts-ignore
+import { defaultTo, replace } from 'ramda'
+import { Field, InjectedFormProps, reduxForm } from 'redux-form'
+
+import { CountryScope } from '@core/types'
+import { BlockchainLoader, Button, HeartbeatLoader, Icon, Text } from 'blockchain-info-components'
+import {
+  FlyoutContainer,
+  FlyoutContent,
+  FlyoutFooter,
+  FlyoutHeader
+} from 'components/Flyout/Layout'
+import DateInputBox from 'components/Form/DateInputBox'
+import FormGroup from 'components/Form/FormGroup'
+import FormItem from 'components/Form/FormItem'
+import TextBox from 'components/Form/TextBox'
+import { model } from 'data'
+import { useCountryList, useUSStateList } from 'hooks'
+import { ageOverEighteen, required, requiredDOB } from 'services/forms'
+
+import { Props as OwnProps, SuccessStateType } from './UserInfoDetails'
+import {
+  Caption,
+  CaptionContainer,
+  ContentWrapper,
+  CustomForm,
+  ErrorText,
+  ErrorTextContainer,
+  Label,
+  SpinnerWrapper
+} from './UserInfoDetails.model'
+
+const { USER_INFO_DETAILS } = model.components.identityVerification
+
+const addTrailingZero = (text: string) => (text.length >= 2 ? text : `0${text}`)
```

Comment:
Another way could be with [padstart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart)
`const addTrailingZero = (text: string) => text.padStart(2, 0);`